### ISSUE TITLE
Add an option to bookmark the news

### ISSUE THREAD

**Issue by nikhilbansal97** [nikhilbansal97/World-Now#4]
The user should be able to bookmark the news. You can use any DB for this but Room will be preferred.

Suggestions for bookmark:
1. The bookmark option should be visible in a overflow menu which will be present in each news item
2. The news items will contain a bookmark icon and clicking on it will save it in db.

**Comment by rosariopfernandes** [comment]
I believe I can take this one.

**Comment by thatfiredev** [comment]
Maybe we could also have a tab where users can see bookmarked news. WDYT?

**Comment by nikhilbansal97** [comment]
@rosariopfernandes Yes, that will also be present. Maybe we can add a navigation drawer that has options to list the latest news and the bookmarks. Can you work on it??

**Comment by nikhilbansal97** [comment]
@rosariopfernandes Are you working on a tab to view the bookmarks??

**Comment by mclintprojects** [comment]
@nikhilbansal97 hey! if no one has taken this up, i'd love to do it. please assign me to it if no one else is working on it.

**Comment by thatfiredev** [comment]
@nikhilbansal97 I'm not working on it. You can assign it to @mclintprojects

**Comment by nikhilbansal97** [comment]
@mclintprojects You can take it up! 👍

**Comment by nikhilbansal97** [comment]
@mclintprojects any updates on progress?
